### ISSUE TITLE
Some constraint file do not load with numpy 2.0

### ISSUE THREAD

**Issue by EGaraldi** [EGaraldi/corecon#32]
### Problem
When using numpy 2.0.0, some file do not load. 

### MWE
```
conda create -n test_latest python=3.12 numpy=2.0 ipython matplotlib=3.9.1
conda activate test_latest
pip install corecon

ipython
Python 3.12.4 | packaged by conda-forge | (main, Jun 17 2024, 10:23:07) [GCC 12.3.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.26.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]:  import corecon as crc
WARNING: Cannot load Mason_et_al_2019.py. Skipping it.
WARNING: Cannot load Lu_et_al_2020.py. Skipping it.
WARNING: Cannot load Palanque-Delabrouille_et_al_2013_fourier.py. Skipping it.
WARNING: Cannot load DAloisio_et_al_2018.py. Skipping it.
WARNING: Cannot load McDonald_et_al_2006.py. Skipping it.
WARNING: Cannot load Croft_et_al_2002.py. Skipping it.
WARNING: Cannot load Palanque-Delabrouille_et_al_2013_likelihood.py. Skipping it.
WARNING: Cannot load Liu_Bordoloi_2021.py. Skipping it.
WARNING: Cannot load Kashikawa_et_al_2015.py. Skipping it.
WARNING: Cannot load Glikman_et_al_2011.py. Skipping it.
WARNING: Cannot load McGreer_et_al_2013.py. Skipping it.
WARNING: Cannot load Ross_et_al_2013.py. Skipping it.
WARNING: Cannot load Jiang_et_al_2016.py. Skipping it.
WARNING: Cannot load Yang_et_al_2016.py. Skipping it.
WARNING: Cannot load Willott_et_al_2010.py. Skipping it.
WARNING: Cannot load Giallongo_et_al_2019.py. Skipping it.
WARNING: Cannot load Giallongo_et_al_2015.py. Skipping it.
WARNING: Cannot load Atek_et_al_2018.py. Skipping it.
WARNING: Cannot load Oesch_et_al_2014.py. Skipping it.
WARNING: Cannot load Morishita_et_al_2018.py. Skipping it.
WARNING: Cannot load Atek_et_al_2015.py. Skipping it.
WARNING: Cannot load Finkelstein_et_al_2015.py. Skipping it.
WARNING: Cannot load Bouwens_et_al_2017.py. Skipping it.
WARNING: Cannot load Castellano_et_al_2016.py. Skipping it.
WARNING: Cannot load Oesch_et_al_2018.py. Skipping it.
WARNING: Cannot load Bouwens_et_al_2022.py. Skipping it.
WARNING: Cannot load Bowler_et_al_2015.py. Skipping it.
WARNING: Cannot load Rorai_et_al_2018.py. Skipping it.
WARNING: Cannot load Meyer_et_al_2019.py. Skipping it.
WARNING: Cannot load Kakiichi_et_al_2018.py. Skipping it.
WARNING: Cannot load Roberts-Borsani_et_al_2022.py. Skipping it.
WARNING: Cannot load Roberts-Borsani_et_al_2022.py. Skipping it.
WARNING: Cannot load Bouwens_et_al_2016.py. Skipping it.
```

**Comment by EGaraldi** [comment]
As a temporary workaround, I will set numpy<2 in the requirements

**Comment by EGaraldi** [comment]
Fixed in f67418ee8d900da6ba59174bd268d19f20a06b5b
